src/runWebPackSync.js

```diff
@@ -11,7 +11,7 @@ export default ({ path, configPath, config, verbose }) => {
 
   const webPackPath = require.resolve('webpack/bin/webpack');
   const rnd = `${(new Date()).getTime()}_${Math.round(1000000 * Math.random())}`;
-  const outPath = join(config.output.path || DEFAULT_OUTPUT_PATH, `.webpack.res.${rnd}.js`);
+  const outPath = join( (config.output && config.output.path) || DEFAULT_OUTPUT_PATH, `.webpack.res.${rnd}.js`);
```

Comment:
Also eslint `14:1   error  Line 14 exceeds the maximum line length of 100  max-len` ;-)